crates/turbopack-ecmascript/src/chunk/mod.rs

```diff
@@ -751,10 +766,13 @@ impl<'a> HmrUpdateEntry<'a> {
     fn new(entry: &'a EcmascriptChunkContentEntry, chunk_path: &str) -> Self {
         HmrUpdateEntry {
             code: entry.source_code(),
-            map: entry
-                .code
-                .has_source_map()
-                .then(|| format!("{}.{}.map", chunk_path, entry.id.to_truncated_hash())),
+            map: entry.code.has_source_map().then(|| {
+                format!(
+                    "/__turbopack_sourcemap__/{}.map?id={}",
+                    chunk_path,
+                    entry.id.to_truncated_hash()
```

Comment:
I guess we don't need to hash the id anymore. We could just send the full id. Urlencoded

Reply:
Done.